### PR TITLE
python3Packages.sqlalchemy: 1.4.19 -> 1.4.20

### PR DESCRIPTION
**Pull request by mweinelt**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/greenlet/default.nix [hunk 1]
```diff
@@ -16,11 +17,18 @@ buildPythonPackage rec {
...
-  propagatedBuildInputs = [ six ];
+  checkPhase = ''
+    runHook preCheck
+    ${python.interpreter} -m unittest discover -v greenlet.tests
+    runHook postCheck
+  '';
 
-  meta = {
+  meta = with lib; {
     homepage = "https://pypi.python.org/pypi/greenlet";
```
Comment (dotlambda) [inline pkgs/development/python-modules/greenlet/default.nix:27]: https://greenlet.readthedocs.io/ or https://github.com/python-greenlet/greenlet

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by dotlambda** [comment]
Btw, `dontUseSetuptoolsCheck = true` shouldn't be needed anymore.

**Comment by mweinelt** [comment]
Fixed

**Review by jonringer — APPROVED** [review]
LGTM

```
https://github.com/NixOS/nixpkgs/pull/128630

1 package built:
python3Packages.pandas
```

**Pull request merged**